JitsiConference.ts

```diff
@@ -1716,6 +1717,26 @@ export default class JitsiConference extends Listenable {
         return this.rtc.getForwardedSources();
     }
 
+    /**
+     * Sets the audio subscription mode for the local user.
+     *
+     * @param {IReceiverAudioSubscriptionMessage} message - The audio subscription mode to set.
+     * @returns {void}
+     */
+    setAudioSubscriptionMode(message: IReceiverAudioSubscriptionMessage): void {
+        this.qualityController.audioController.setAudioSubscriptionMode(message);
+    }
+
+    /**
+     * Sets deafened status for the local user.
+     *
+     * @param {boolean} isDeafened - Whether the user should stop receiving remote audio.
+     * @returns {void}
+     */
+    setIsDeafened(isDeafened: boolean): void {
```

Comment:
I'm a bit hesitant to expose this as a public API since it dierges from widespread use. (in Discord). In Discord, deafening also mutes you.

Now, if we are to mute the user, we shall do that at the application layer, since it may need to collaborate with other muting operations (aka the authority).

Reply:
Why wouldn't it be? From the LJM perspective, it should be a public API so that the consumers of the library can call it and implement the deafened feature however they want. We could rename it to something else more meaningful perhaps?

Reply:
That's what I meant, that with the current name I think the expectation would be that it does what discord does.

I don't have a good name suggestion, why not leave it at setAudioSubscriptionMode and not provide this higher level one just yet?

Reply:
Will `muteRemoteAudio` work? I feel this is more intuitive if someone were to implement this functionality in their application rather than figuring the various subscription modes that the bridge supports.

Reply:
LJM is supposed to be the low level library, so IMHO we better start with less API surface and enlarge later if necesssary.

Won't oppose landing under the new name though.